tldextract/tldextract.py

```diff
@@ -187,8 +187,15 @@ def __call__(self, url):
             .rstrip(".")
 
         is_punycode = netloc.startswith('xn--') or '.xn--' in netloc
+        # Some IDN urls might generate an error
         if is_punycode:
-            netloc = codecs.decode(netloc.encode('ascii'), 'idna')
+            try:
+                netloc = codecs.decode(netloc.encode('ascii'), 'idna')
+            except UnicodeError:
+                LOG.warn('UnicodeError when decoding netloc ' + netloc)
```

Comment:
Uhm, no it's not. I'm used to log every "unintended message" but I agree with you. So I'll remove it.